Toolchain/BuildIt.sh

```diff
@@ -198,6 +219,27 @@ pushd "$DIR/Tarballs"
         echo "Skipped downloading gcc"
     fi
 
+    if [ "$ARCH" = "aarch64" ]; then
+        if [ -d ${GDB_NAME} ]; then
+            rm -rf "${GDB_NAME}"
+            rm -rf "$DIR/Build/$ARCH/$GDB_NAME"
+        fi
+        echo "Extracting GDB..."
+        tar -xzf ${GDB_PKG}
+
+        pushd ${GDB_NAME}
+            if [ "$git_patch" = "1" ]; then
+                git init > /dev/null
+                git add . > /dev/null
+                git commit -am "BASE" > /dev/null
+                git apply "$DIR"/Patches/gdb.patch > /dev/null
```

Comment:
Nit: Do we need a local gdb.patch? I don't see one in this PR, but I'm not sure if you forgot to `git add` it or if this is just here because the other packages use these commands and you copied an existing command (in case we need a patch later). If we don't need a patch, I'd omit the patching logic for now.

Reply:
Oh my god i totally forgot to commit the patch file, good catch!

Reply:
Fixed